next/api/resources/retired.py

```diff
@@ -0,0 +1,20 @@
+from flask import Flask, request, send_file
```

Comment:
[This] (https://github.com/liamim/NEXT/blob/86e4f2159b03e3ce85d71d102f5e9427474e4c91/next/dashboard/dashboard.py#L42) (the dashboard page) endpoint, correct?